packages/owlbot-bootstrapper/cli/run-trigger.ts

```diff
@@ -21,13 +21,14 @@ import {CliArgs} from './run-trigger-command';
 const COMMON_CONTAINER =
   'us-docker.pkg.dev/owlbot-bootstrap-prod/owlbot-bootstrapper-images/owlbot-bootstrapper:latest';
 const DIRECTORY_PATH = '/workspace';
-const MONO_REPO_PATH = DIRECTORY_PATH;
+const MONO_REPO_DIR = DIRECTORY_PATH;
 const SERVICE_CONFIG_PATH = `${DIRECTORY_PATH}/serviceConfig.yaml`;
 const INTER_CONTAINER_VARS_PATH = `${DIRECTORY_PATH}/interContainerVars.json`;
 
 export async function runTrigger(
   argv: CliArgs,
   cb: CloudBuildClient,
+  monoRepoNameAndOrg: {monoRepoName: string; monoRepoOrg: string},
```

Comment:
This type could be re-used. Additionally, the `monoRepo` part could be redundant.

Consider a type like:

```ts
interface GitHubRepository {
  owner: string;
  name: string;
}
```